lldb/source/Commands/CommandObjectCommands.cpp

```diff
@@ -1644,8 +1647,9 @@ class CommandObjectCommandsScriptAdd : public CommandObjectParsed,
       llvm::Error llvm_error =
           m_container->LoadUserSubcommand(m_cmd_name, new_cmd_sp, m_overwrite);
       if (llvm_error)
-        result.AppendErrorWithFormat("cannot add command: %s", 
-                                     llvm::toString(std::move(llvm_error)).c_str());
+        result.AppendErrorWithFormat(
+            "cannot add command: %s",
+            llvm::toString(std::move(llvm_error)).c_str());
```

Comment:
revert non related whitespace change

Reply:
No, it is actually required by the formatter reported by github: 
https://github.com/llvm/llvm-project/actions/runs/7789051206/job/21239805569?pr=80375